commit ebf3a0398f18eab67d2ba25e6a10b41ff140f6a4
Author: Duncan Bellamy <dunk@denkimushi.com>
Date:   Fri May 21 06:30:10 2021 +0100

    rgw/rgw_string.h: add missing includes for alpine and boost 1.75
    alpine needs: string, stdexcept
    
    Fixes: https://tracker.ceph.com/issues/50924
    Signed-off-by: Duncan Bellamy <dunk@denkimushi.com>

diff --git a/src/rgw/rgw_string.h b/src/rgw/rgw_string.h
index 257daa9c1fe..90e64f98a25 100644
--- a/src/rgw/rgw_string.h
+++ b/src/rgw/rgw_string.h
@@ -8,6 +8,8 @@
 #include <stdlib.h>
 #include <limits.h>
 #include <string_view>
+#include <string>
+#include <stdexcept>
 
 #include <boost/container/small_vector.hpp>
 
